docker/sedona-spark-jupyterlab/sedona-jupyterlab.dockerfile

```diff
@@ -29,8 +28,7 @@ ARG spark_extension_version=2.11.0
 
 # Set up envs
 ENV SHARED_WORKSPACE=${shared_workspace}
-ENV SPARK_HOME /opt/spark
-RUN mkdir ${SPARK_HOME}
+ENV SPARK_HOME /usr/local/lib/python3.10/dist-packages/pyspark
```

Comment:
@jiayuasu I think this might break if/when the base image changes the Python version